tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "module": "es2020",
+    "module": "node16",
```

Comment:
Why did we downgrade this to node16

Reply:
ah, this was accidentally pushed, as vscode suggested it